addons/sys_core/fnc_processRadioSpeaker.sqf

```diff
@@ -22,6 +22,7 @@ params["_unit","_playerRadios"];
 
 private _radioId = _unit getVariable QGVAR(currentSpeakingRadio);
 if(_radioId == "") exitWith { false };
+//if(ACRE_LISTENER_DIVE == 1) exitWith { false };
```

Comment:
Done in https://github.com/IDI-Systems/acre2/pull/66/commits/36ef2b39ebddd695a285e1ddc7ffd77e22dd1dbb